lively.morphic/morph.js

```diff
@@ -270,8 +270,7 @@ export class Morph {
         defaultValue: pt(0, 0),
         get () {
           let layout;
-          if (layout = this._askLayoutForBounds) {
-            this._askLayoutForBounds = false;
+          if (layout = this.renderingState.cssLayoutToMeasureWith) {
```

Comment:
This results in an assignment and afterwards a check if the assigned value was truthy?